dist/goaccess/obs-goaccess-last-month.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+# Generates goaccess HTML data from last months rotated apache access logs
+# Runs from cron.d on every first of the month
+
+NOW_MONTH=$(date +%Y-%m)
+PREVIOUS_MONTH=$(date -d "$nowmonth-15 last month" '+%Y%m')
```

Comment:
```suggestion
CURRENT_MONTH=$(date +%Y-%m)
PREVIOUS_MONTH=$(date -d "$CURRENT_MONTH-15 last month" '+%Y%m')
```
There is some inconsistency here...